controllers/articles.controllers.js

```diff
@@ -14,4 +14,13 @@ function getArticles(req, res, next) {
     })
 }
 
-module.exports = {getArticleById, getArticles};
\ No newline at end of file
+function patchArticleById(req, res, next) {
+    updateArticleById(req.params, req.body).then(article => {
+        res.status(200).send({article});
+    }).catch(err => {
+        console.log(err);
+        next(err);
```

Comment:
don't forget to clean up old console logs once you're done 🧹 
Really good controller function